src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,289 @@
+
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on MySQL card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+    <Step>
+        ## Install our Apache agent
+
+        Install Apache agent to know more about TCP connections, DNS lookup , HTTPS, SSL, Server and Worker status.
+        Follow Apache documentation: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/
+
+        From one.newrelic.com:
+                    
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on Apache card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+
+    <Step>
+        ## Install our Elasticsearch 
+        
+        Install Elasticsearch agent , if one is using Magento version above 2.4, one needs to install an ElasticSearch application and agent to monitor that application.You can monitor metrics like Cluster status and Current shards.
+        docs:https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration/
+        Reference:https://magentip.com/install-magento-2-4-x-on-ubuntu-with-elasticsearch/
+
+        From one.newrelic.com:
+                    
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on Elasticsearch card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+
+    <Step>
+        ## Set up synthetic monitoring
+
+        Synthetic monitoring allows you to monitor your site vailability and downtime-related metrics.
+
+        1. Click Add data
+        2. Click Simulate traffic
+        3. Click on Synthetic monitoring card
+        4. Select monitor type i.e, Availability and Page load 
+        5. Configure monitor by giving Name, Tags, URL and Period
+        6. Select locations 
+        7. Save monitor
+
+        </Step>
+    
+
+    <Step>
+        ## Install AWS 
+        
+        Install AWS in order to get metrics like Certificates per account, Groups per account, MFA devices per account, Roles per account and Users per account.
+
+        From one.newrelic.com:
+            
+        1. Click Add data
+        2. Click Amazon Web Services
+        3. Click on Integrate your AWS account card
+        4. Select an account
+        5. Select Amazon Web Services you want to monitor with New Relic.
+        Here is the reference docs for AWS IAM :https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-iam-monitoring-integration/
+        </Step>
+    
+
+    <Step>
+        ## Configure our Flex integration
+
+        1. Create an nri-flex configuration file:
```

Comment:
Is NR Flex also optional since it comes bundled with the infra agent (and we say the infra agent is optional)?

Reply:
My apology for writing infra agent as optional, its required